CovertInfiltration/Src/CovertInfiltration/Classes/X2InfiltrationModTemplateManager.uc

```diff
@@ -0,0 +1,27 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  NotSoLoneWolf
+//  PURPOSE: This class contains functions to control and
+//           retrieve X2InfiltrationMod templates.
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+
+class X2InfiltrationModTemplateManager extends X2DataTemplateManager;
+
+static function X2InfiltrationModTemplateManager GetInfilTemplateManager()
+{
+    return X2InfiltrationModTemplateManager(class'Engine'.static.GetTemplateManager(class'X2InfiltrationModTemplateManager'));
+}
+
+function X2InfiltrationModTemplate GetInfilTemplateFromItem(X2ItemTemplate ItemTemplate)

```

Comment:
Please take the item template name, not the template itself. The reason is that if:

* you have the template then to get the name you just do `Template.DataName`
* you have the name then to get the template you need the manager and then the template from there